pkg/kube/controllers/extendedjob/query.go

```diff
@@ -59,3 +65,29 @@ func (q *QueryImpl) MatchState(eJob ejv1.ExtendedJob, podState ejv1.PodState) bo
 	}
 	return false
 }
+
+// IsJobAlreadyExists checks if the job is already created for the pod event match with exjob
+func (q *QueryImpl) IsJobAlreadyExists(ctx context.Context, eJob ejv1.ExtendedJob, podUID types.UID, r *TriggerReconciler) (bool, error) {
```

Comment:
rename to `JobExists` please

Reply:
Also, the comment should be:
```
// JobExists returns true if the Job matching the ExtendedJob has already been created
```

please also rename the `r` parameter to `triggerReconciler`